modules/_labs/combobox/react/lib/Combobox.tsx

```diff
@@ -360,9 +397,76 @@ export default class Combobox extends React.Component<ComboboxProps, ComboboxSta
     });
   };
 
+  createList = () => {
```

Comment:
Could this be broken out into a functional component? This file is getting awfully long

Reply:
I think I am going to open a PR to make combobox functional first. Just to make things consistent in the folder and easier to grok